website/src/components/ContextMessages.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@chakra-ui/react";
+import { Message } from "./Messages";
+
+export const ContextMessages = ({ messages }: { messages: Message[] }) => {
+  return (
+    <Box className="flex flex-col gap-1">
+      {messages.map((message, i) => {
+        return (
+          <Box key={i}>
+            <span>{message.is_assistant ? "Assistant: " : "User: "}</span>
```

Comment:
Later this could look good with a Chakra label or some icon.

Reply:
Complete agree. Got something basic working, and left the design for someone who's stronger in that area.